utility/mm2_update-master-directory-list

```diff
@@ -52,21 +53,32 @@ class MasterFilter(logging.Filter):
         record.category = cname
         return True
 
-def make_file_details_from_checksums(session, diskpath, relativeDName, D):
+def make_file_details_from_checksums(session, config, D, files=None):
     def _parse_checksum_file(relativeDName, checksumlen):
         r = {}
         try:
-            f = open(os.path.join(diskpath, relativeDName),  'r')
+            f = open(os.path.join(config.get('UMDL_PREFIX', ''), relativeDName),  'r')
```

Comment:
Thanks for all the reviews, could you give me an example what you mean to use a context manager. Would

    with open(os.path.join(config.get('UMDL_PREFIX', ''), relativeDName),  'r') as f:

be enough? Then the close() could also be removed?